state/state_test.go

```diff
@@ -257,7 +257,7 @@ func TestValidatorSimpleSaveLoad(t *testing.T) {
 	assert.Nil(err, "expected no err at height 2")
 	assert.Equal(v.Hash(), state.NextValidators.Hash(), "expected validator hashes to match")
 
-	// Increment height, save; should be able to load for next & next next height.
+	// Increment height, save; should be able to load for next & next height.
```

Comment:
I think this one was intentional

```suggestion
	// Increment height, save; should be able to load for next & one after next height.
```